### DISCUSSION TITLE
What is the difference between Cypress dashboard parallel test and CI parallel test?

### DISCUSSION THREAD

**Discussion by baeharam** [cypress-io/cypress#14216] [CI setup] [answered]
I'm using CircleCI and it provides feature to run parallel tests ( = parallel jobs ). Then, how cypress dashboard different from CI parallel test? I read dashboard docs, but couldn't find special reason to use dashboard. Can you explain? Thanks

**Comment by baeharam** [comment]
@bahmutov Can you answer for this question?

**Comment by jennifer-shehane** [comment, marked as answer]
I'd suggest reading through our Parallelization guide. I think it summarizes pretty well how Cypress orchestrates parallel tests to work with your CI provider (which allows you to run multiple machines in parallel). https://on.cypress.io/parallelization

**Comment by baeharam** [comment]
@jennifer-shehane Thanks. I read that docs and understood that the only difference is load-balancing using timing data. Is it right?

**Reply by bahmutov** [reply]
Yes, using timings to order specs, plug groups, plus the simplicity of the entire system. Pretty much every CI system works right out of the box with simple `--parallel` flag.
